source/IntegrationTests/Behaviours/IncomingRequests/GivenAggregatedMeasureDataV2RequestWithDelegationTests.cs

```diff
@@ -800,92 +876,113 @@ await ThenRejectRequestAggregatedMeasureDataDocumentIsCorrect(
                 InstantPattern.General.Parse("2024-07-01T14:57:09Z").Value,
                 ReasonCode.FullyRejected.Code,
                 TransactionId.From("12356478912356478912356478912356478"),
-                "E17",
-                expectedReasonMessage));
+                expectedErrorCode,
+                expectedErrorMessage));
     }
 
     /// <summary>
     /// Even though an actor has delegated his requests to another actor, he should still
     /// be able to request and receive his own data
     /// </summary>
-    [Theory(Skip = "not updated yet")]
-    [MemberData(nameof(DocumentFormatsWithRoleCombinationsForNullGridArea), MemberType = typeof(GivenAggregatedMeasureDataRequestWithDelegationTests))]
-    public async Task AndGiven_DelegationInOneGridArea_AndGiven_OriginalActorRequestsOwnDataWithDataInThreeGridAreas_When_OriginalActorPeeksAllMessages_Then_OriginalActorReceivesThreeNotifyAggregatedMeasureDataDocumentWithCorrectContent(DocumentFormat incomingDocumentFormat, DocumentFormat peekDocumentFormat, ActorRole delegatedFromRole, ActorRole delegatedToRole)
+    [Theory]
+    [MemberData(
+        nameof(DocumentFormatsWithRoleCombinationsForNullGridArea),
+        MemberType = typeof(GivenAggregatedMeasureDataV2RequestWithDelegationTests))]
+    public async Task AndGiven_DelegationInOneGridArea_AndGiven_OriginalActorRequestsOwnDataWithDataInTwoGridAreas_When_OriginalActorPeeksAllMessages_Then_OriginalActorReceivesThreeNotifyAggregatedMeasureDataDocumentWithCorrectContent(
```

Comment:
```suggestion
    public async Task AndGiven_DelegationInOneGridArea_AndGiven_OriginalActorRequestsOwnDataWithDataInTwoGridAreas_When_OriginalActorPeeksAllMessages_Then_OriginalActorReceivesTwoNotifyAggregatedMeasureDataDocumentWithCorrectContent(
```
Two not three notify

This tests what the other test was supposed to?